### DISCUSSION TITLE
Saving changes made in Developer Debug Mode.

### DISCUSSION THREAD

**Discussion by lgs777** [lllyasviel/Fooocus#2147] [Q&A] [answered]
How do I save some of the settings I changed in Developer Debug Mode? I don't want to have to adjust the values every time I run it again, and was wondering if there is a guide for this.

**Comment by Irmagon** [comment, marked as answer]
Hi
Try create preset with u needed settings values

**Reply by mashb1t** [reply]
@lgs777 this is the official solution until https://github.com/lllyasviel/Fooocus/pull/1940 is merged. Then you can simply drag&drop a generated image into Fooocus and it will apply all changes (except for img2img) to settings automatically.

**Reply by lgs777** [reply]
@mashb1t @Irmagon Thank you!
